benchmark/utils_sr.js

```diff
@@ -0,0 +1,167 @@
+// Benchmark for Super Resolution models
+class SRBenchmark extends Benchmark {
+  constructor(modelName, backend, iterations) {
+    super(...arguments);
+    this.modelInfoDict = getModelInfoDict(superResolutionModels, modelName);
+    this.model = null;
+    this.inputTensor = null;
+    this.outputTensor = null;
+    this.outputElement = document.getElementById('superCanvas');
+    this.inCanvas = document.createElement('canvas');
```

Comment:
@cuiyanx  Those 4 members inCanvas/inCtx  / outCanvas/outCtx aren't needed, they can be declared when using.
 Please move inCanvas/inCtx into function setInputOutput(), and move outCanvas/outCtx into function drawOutput(canvas).